affirm_example/templates/index.html

```diff
@@ -113,10 +123,17 @@ <h3>Complete your checkout</h3>
 
   function openCheckout(input) {
     var priceInput = document.getElementById('item-price');
+    var merchantEl = document.getElementById('merchant');
+    var country = document.getElementById('country').value;
     var price = Math.round(Number(priceInput.value) * 100);
+
+    if (merchantEl && !merchantEl.value) {
```

Comment:
curious when would this happen?

Reply:
this is an edge case but possible if `PUBLIC_API_KEY` and `SECRET_API_KEY` are set, but the `MERCHANTS` config does not have that merchant in it

Reply:
okay, so I guess we should probably make sure this won't happen while seeding the data.
Otherwise the e2e tests might break.

Reply:
actually, maybe we always select the first merchant no matter what? so we don't have to deal with this edge case?

and while we are seeding the data, we could use the default keys for the first merchant.

This would be similar to what you had, but this would give us potential to eventually deprecate the `config['AFFIRM']['PUBLIC_API_KEY']` config if needed in the future

Reply:
updated - [72f40f8](https://github.com/Affirm/affirm-example-python/pull/32/commits/72f40f84df8f189cefc63ea69061b2af1103f46c)